### PR TITLE
[Multimodal] MMEFT: Add multilabel examples for MMEFT

### PR DESCRIPTION
**Pull request by vivek-dani**
# Description
Add sdk and cli examples for Multilabel classification problem for MMEFT model

# Checklist

- [x] I have read the contribution guidelines.
  - [General](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
  - [SDK](https://github.com/Azure/azureml-examples/blob/main/sdk/CONTRIBUTING.md)
  - [CLI](https://github.com/Azure/azureml-examples/blob/main/cli/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] Pull request includes test coverage for the included changes.
- [x] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure/azureml-examples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

File: cli/foundation-models/system/finetune/multimodal-classification/multilabel-classification/mmeft-chxray-multilabel-classification.sh [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+# Scoring file
+sample_request_data="./sample_request_data.json"
+
+# finetuning job parameters
+finetuning_pipeline_component="multimodal_classification_pipeline"
+# Training settings
+process_count_per_instance=$gpus_per_node # set to the number of GPUs available in the compute
+
+# 1. Install dependencies
+pip install azure-ai-ml==1.8.0
```
Comment (rjaincc) [inline cli/foundation-models/system/finetune/multimodal-classification/multilabel-classification/mmeft-chxray-multilabel-classification.sh:42]: We can use latest version

File: sdk/python/foundation-models/system/finetune/multimodal-classification/multilabel-classification/mmeft-chxray-multilabel-classification.ipynb [hunk 2]
```diff
@@ -0,0 +1,1031 @@
...
+    "Before starting off, if you are running the notebook on Azure Machine Learning Studio or running first time locally, you will need the following packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! pip install azure-ai-ml==1.8.0\n",
```
Comment (rjaincc) [inline sdk/python/foundation-models/system/finetune/multimodal-classification/multilabel-classification/mmeft-chxray-multilabel-classification.ipynb:46]: Lets use latest versions.

File: sdk/python/foundation-models/system/finetune/multimodal-classification/multilabel-classification/mmeft-chxray-multilabel-classification.ipynb [hunk 3]
```diff
@@ -0,0 +1,1031 @@
...
+    "print(uri_folder_data_asset.path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 4.3. Update image url in dataset\n",
+    "\n",
+    "[csv_processor.py](../utils/csv_processor.py) script updates the path to images in .csv files, from local path to path in AML datastore, where we uploaded the images in Step 2.2"
```
Comment (rjaincc) [inline sdk/python/foundation-models/system/finetune/multimodal-classification/multilabel-classification/mmeft-chxray-multilabel-classification.ipynb:343]:
```suggestion
    "[csv_processor.py](../utils/csv_processor.py) script updates the path to images in .csv files, from local path to path in AML datastore, where we uploaded the images in Section 2.2"
```

File: sdk/python/foundation-models/system/finetune/multimodal-classification/multilabel-classification/mmeft-chxray-multilabel-classification.ipynb [hunk 4]
```diff
@@ -0,0 +1,1031 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
```
Comment (rjaincc) [inline sdk/python/foundation-models/system/finetune/multimodal-classification/multilabel-classification/mmeft-chxray-multilabel-classification.ipynb:1010]: Clean up metadata in all the ipynb files before merging this

**Review by rjaincc — APPROVED** [review]
Approving with minor comments. 

Clean up metadata in all the ipynb files before merging this

File: cli/foundation-models/system/finetune/multimodal-classification/multilabel-classification/prepare_data.py [hunk 5]
```diff
@@ -0,0 +1,297 @@
...
+    :type ml_client: MLClient
+    :param dataset_parent_dir: dataset parent directory
+    :type dataset_parent_dir: str
+
+    """
+    # Create data folder if it doesnt exist.
+    os.makedirs(dataset_parent_dir, exist_ok=True)
+
+    # Download data
+    download_url = "https://automlresources-prod.azureedge.net/datasets/ChXray.zip"
```
[the remark this answers is not in the document]
Comment (vivek-dani) [reply cli/foundation-models/system/finetune/multimodal-classification/multilabel-classification/prepare_data.py:128]: this is working url. Phani just added to CDN sometime back

File: cli/foundation-models/system/finetune/multimodal-classification/multilabel-classification/prepare_data.py [hunk 6]
```diff
@@ -0,0 +1,297 @@
...
+from os.path import basename, dirname, join
+from zipfile import ZipFile
+
+from azure.identity import DefaultAzureCredential
+from azure.ai.ml import MLClient
+from azure.ai.ml.entities import Data
+from azure.ai.ml.constants import AssetTypes
+
+
+def create_ml_table_file(filename) -> str:
```
[the remark this answers is not in the document]
Comment (vivek-dani) [reply cli/foundation-models/system/finetune/multimodal-classification/multilabel-classification/prepare_data.py:17]: I have added in onenote and work-item ticket. This being public repo, I am not adding it here

File: cli/foundation-models/system/finetune/multimodal-classification/multilabel-classification/mmeft-chxray-multilabel-classification.sh [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+
+# Check if scoring data file exists
+if [ -f $sample_request_data ]; then
+    echo "Invoking endpoint $online_endpoint_name with $sample_request_data\n\n"
+else
+    echo "Scoring file $sample_request_data does not exist"
+    exit 1
+fi
+
+az ml online-endpoint invoke --name $online_endpoint_name --request-file $sample_request_data $workspace_info || {
```
[the remark this answers is not in the document]
Comment (vivek-dani) [reply cli/foundation-models/system/finetune/multimodal-classification/multilabel-classification/mmeft-chxray-multilabel-classification.sh:160]: aligning with vision examples. We don't have separate examples for multilabel-image-classification

**Review by shubhamiit — APPROVED** [review]

**Pull request merged**